src/HomePage/Home.js

```diff
@@ -0,0 +1,143 @@
+import React, { useState } from 'react'
+import { getFirestore, doc, updateDoc } from 'firebase/firestore'
+import { app } from '../Fire'
+
+const data = [
+  {
+    current: 1,
+    one: 2,
+    two: 9,
+    path: 'images/screen1.png'
+  }, {
```

Comment:
You might want to take these closing braces on the next line to keep your code consistent.

Reply:
Done